app/src/main/AndroidManifest.xml

```diff
@@ -24,7 +24,8 @@
             android:exported="true"
             android:screenOrientation="portrait"
             android:theme="@style/Theme.App.Starting"
-            android:windowSoftInputMode="adjustResize">
+            android:windowSoftInputMode="adjustResize"
+            android:usesCleartextTraffic="true">
```

Comment:
What is the news regarding this, are we keeping it?

Reply:
right, I had already removed this locally and will push it with some other changes!